src/middlewares/__tests__/item-authorization.test.js

```diff
@@ -0,0 +1,95 @@
+const itemAuthorization = require('../item-authorization');
+const Item = require('../../models/item');
+
+const trueOwnerId = '123456';
+const falseOwnerId = '654321';
+const correctItemId = 'correctId';
+const incorrectItemId = 'incorrectId';
+const item = {
+  _id: 'correctId',
+  owner: '123456',
+};
+const res = {
+  status: jest.fn().mockReturnThis(),
+  json: jest.fn().mockReturnThis(),
+};
+jest.mock('../../models/item', () => ({
+  findById: jest.fn().mockImplementation((id) => {
+    // eslint-disable-next-line no-underscore-dangle
+    if (id === item._id) {
+      return Promise.resolve(item);
+    }
+    return Promise.resolve(null);
+  }),
+}));
+
+afterAll(() => {
+  jest.clearAllMocks();
+});
+describe('itemAuthorization function middleware ', () => {
+  it('should authorize user when his id equals to item owner id', async () => {
```

Comment:
Try to avoid mentioning implementation logic in test statements.
So here you can rephrase to `Should authorize when requesting user is the owner of requested item`.

Reply:
Check the other test statements too and see if there's any rephrasing required similar to this.